mappings/net/minecraft/world/gen/surfacebuilder/SurfaceBuilder.mapping

```diff
@@ -30,16 +30,20 @@ CLASS net/minecraft/class_6724 net/minecraft/world/gen/surfacebuilder/SurfaceBui
 		ARG 3 seaLevel
 		ARG 4 seed
 		ARG 6 randomProvider
-	METHOD method_39102 (Lnet/minecraft/class_6557;IIILnet/minecraft/class_5539;)V
+	METHOD method_39102 buildBadlandsPillar (Lnet/minecraft/class_6557;IIILnet/minecraft/class_5539;)V
```

Comment:
Isn't this called "hoodoo"?

Reply:
Perhaps, I named this from the noises it uses (`badlandsPillarNoise` and `badlandsPillarRoofNoise`, which in turn are based on exposed registry names)